docs/it/remote-working.md

```diff
@@ -0,0 +1,91 @@
+
+I vantaggi di lavorare viaggiando sono molteplici:
+
+- esplorare nuovi paesi senza dover prendere ferie;
+- vivere per lunghi periodi in paesi con costo della vita più basso;
+- cercare un clima più adatto alle proprie abitudini - periodi caldi d'inverno, freschi d'estate, ad esempio;
+- fare nuove esperienze fuori dall'orario di lavoro.
+
+Ovviamente, questo può creare una visione idealistica del nomade digitale, ma non sono poche le difficoltà:
+
+- necessità di adattarsi ad ogni nuovo spostamento;
+- avere a che fare con fuso orari anche molto diversi da quello della propria azienda;
+- problemi di socialità se si viaggia in solitaria;
+- problemi tecnici, come connessioni scadenti, che possono causare problemi nel lavorare.
+
+Tutti questi fattori sono assolutamente da tenere in considerazione se si valuta di partire per lavorare da altri paesi, e il consiglio per minimizzare l'impatto del viaggio è concedersi dei giorni senza lavorare appena arrivati in un nuovo paese, per poter verificare le modalità di lavoro che ci saranno, scegliere paesi che non richiedano di lavorare in orari assurdi, prediligere località _nomad friendly_ per conoscere altri viaggiatori che lavorano e avere più di una soluzione per quanto riguarda la propria postazione di lavoro nomade.
+
+Esistono molte community online di altri nomadi digitali e siti che permettono di valutare le mete preferibili per iniziare, uno su tutti [Nomad list](https://nomadlist.com/), ma in assoluto viaggiare lavorando è una pratica sempre più diffusa che può davvero cambiare scenario alle giornate di lavoro.
+
+## Programmare da remoto funziona davvero?
+
+Se il lavoro da remoto porta molti vantaggi sulla gestione della proprio vita privata, ci sono però aspetti tecnici che infiammano le discussioni tra noi developer, con molte criticità che possono presentarsi con questa modalità di lavoro.
+
+La più evidente riguarda la formazione delle figure junior, che entrando in un'azienda completamente da remoto non hanno la possibilità di lavorare letteralmente _fianco a fianco_ a figure senior e perdono la possibilità di apprendere velocemente dall'esperienza, ritrovandosi isolati in una fase molto importante della loro formazione.
+
+In realtà a questo si può in parte sopperire con uno sforzo maggiore dell'azienda e del resto del team a implementare un processo di on-boarding strutturato, sessioni dedicate esclusivamente alla formazione e permettere a dev con meno seniority di poter facilmente avere supporto in caso di problematiche, tecniche e non.
+
+Un'altra criticità riguarda la difficoltà di coordinare il lavoro, con meeting fatti in videochiamata poco produttivi rispetto a sessioni di brain storming di persona e più in generale poca coesione all'interno del team rispetto ad un gruppo di developer che lavorano nella stessa stanza.
```

Comment:
Pensi sia il posto e il momento giusto per parlare anche di questioni strettamente previdenziali/assicurative? Ad esempio alcune aziende non permettono ai propri dipendenti di lavorare oltre una certa quantità di giorni all'estero

Reply:
Mancava questo punto, grazie. L'ho inserito in quella parte di capitolo, dimmi cosa ne pensi e correggi pure se necessario :). Ho dovuto anche rimuovere le correzioni alle liste che mi avevi fatto perché mi davano errore nel format:check (* al posto di -)